Backend/Backend-Read/backend-read_products/components/Product/index.js

```diff
@@ -24,6 +24,18 @@ export default function Product() {
       <p>
         Price: {data.price} {data.currency}
       </p>
+      <article>
+        <h4>Reviews</h4>
+        <h5>
+          <q>{data.reviews[0].title}</q>
```

Comment:
I am not sure what it shows ... but it must be only one value, right? Because it is the first element of the index. I used map in order to get all reviews in case there are more. Also in the next values down there:

Reply:
Yeah I looked again and it only works for some of the fish. I updated the code to map over the reviews like we discussed yesterday. Thanks, @TS-Severin!